plugin/__tests__/fixtures/buildGradleFiles.ts

```diff
@@ -0,0 +1,164 @@
+}
+
+android {
+    ndkVersion rootProject.ext.ndkVersion
+
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+        applicationId 'com.test.withIAP'
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 34
+        versionName "1.16.2"
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()`;
+
+const appBuildGradleWithPlayStoreIAP = `
+apply plugin: "com.android.application"
+
+import com.android.build.OutputFile
+
+def reactNativeArchitectures() {
+    def value = project.getProperties().get("reactNativeArchitectures")
+    return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
+}
+
+android {
+    ndkVersion rootProject.ext.ndkVersion
+
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+missingDimensionStrategy "store", "play"
+        applicationId 'com.test.withIAP'
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 34
+        versionName "1.16.2"
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()`;
+
+const appBuildGradleWithAmazonStoreIAP = `
+apply plugin: "com.android.application"
+
+import com.android.build.OutputFile
+
+def reactNativeArchitectures() {
+    def value = project.getProperties().get("reactNativeArchitectures")
+    return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
+}
+
+android {
+    ndkVersion rootProject.ext.ndkVersion
+
+    compileSdkVersion rootProject.ext.compileSdkVersion
+
+    defaultConfig {
+missingDimensionStrategy "store", "amazon"
+        applicationId 'com.test.withIAP'
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 34
+        versionName "1.16.2"
+        buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled().toString()`;
+
+const appBuildGradleWithBothIAP = `
+apply plugin: "com.android.application"
+
+import com.android.build.OutputFile
+
+def reactNativeArchitectures() {
+    def value = project.getProperties().get("reactNativeArchitectures")
+    return value ? value.split(",") : ["armeabi-v7a", "x86", "x86_64", "arm64-v8a"]
+}
+
+android {
+    ndkVersion rootProject.ext.ndkVersion
+
+    compileSdkVersion rootProject.ext.compileSdkVersion
+flavorDimensions "appstore"
+
+productFlavors {
+  googlePlay {
+    dimension "appstore"
+    missingDimensionStrategy "store", "play"
+  }
+
+  amazon {
+    dimension "appstore"
+    missingDimensionStrategy "store", "amazon"
+  }
+}
+
+    defaultConfig {
+        applicationId 'com.test.withIAP'
+        minSdkVersion rootProject.ext.minSdkVersion
+        targetSdkVersion rootProject.ext.targetSdkVersion
+        versionCode 34
+        versionName "1.16.2"
```

Comment:
Are these arbitrary/ do they matter to the person building an expo app?

Reply:
All files in the `fixtures` directory are used solely for testing and will only be utilized by a developer writing tests.